contrib/exchange-bmc-os-info.init.redhat

```diff
@@ -64,9 +64,9 @@ SCRIPT_NAME=$(basename $0)
 
 RETVAL=0
 
-if [ -f /bin/gettext.sh ]; then
+if [ -f /usr/bin/gettext.sh ]; then
```

Comment:
Although this is redhat-specific, why not search in PATH like I proposed for the other script?